webpack.config.js

```diff
@@ -1,12 +1,18 @@
 const path = require("path");
 const MiniCssExtractPlugin = require("mini-css-extract-plugin");
 const MonacoWebpackPlugin = require("monaco-editor-webpack-plugin");
-const UglifyJsPlugin = require('uglifyjs-webpack-plugin');
+const UglifyJsPlugin = require("uglifyjs-webpack-plugin");
+const CopyPlugin = require("copy-webpack-plugin");
 const { ProvidePlugin } = require("webpack");
 
 const OUTPUT_FOLDER = path.join(__dirname, "build");
 const CODEBASE = path.join(__dirname, "source");
-const isProd = process.env.NODE_ENV === "prod";
+
+const { NODE_ENV, RELEASE_VERSION, MODE } = process.env;
+const isProd = NODE_ENV === "prod";
+
+const urlPrefix = MODE === "website" ? "bitrise_workflow_editor-" : "";
+const publicPath = isProd ? `/${urlPrefix}${RELEASE_VERSION}/` : "";
```

Comment:
this: https://github.com/bitrise-io/bitrise-workflow-editor/blob/master/rails/transformer.rb#L19 had to be duplicated here